yt/visualization/profile_plotter.py

```diff
@@ -209,10 +209,11 @@ class ProfilePlot(object):
     Use set_line_property to change line properties of one or all profiles.
     
     """
+
     x_log = None
     y_log = None
-    x_title = None
-    y_title = None
+    x_title = {}
+    y_title = {}
```

Comment:
These should be set to `None` at class level as they were before. It's fine to leave `x_title` as `None` by default since there is only one x field, but `y_title` should get initialized as an empty dict in the `_initialize_instance` method, in the same way that `y_log` does.

They way you have it here, the dict is shared among all instances of the class. See e.g. http://docs.python-guide.org/en/latest/writing/gotchas/#mutable-default-arguments for an explanation of why this is a problem.

Reply:
Ah, yeah. I see why that is an issue